cpp/include/cudf/detail/utilities/linked_column.hpp

```diff
@@ -35,30 +37,11 @@ struct linked_column_view : public column_view_base {
   linked_column_view(linked_column_view const&) = delete;
   linked_column_view& operator=(linked_column_view const&) = delete;
 
-  linked_column_view(column_view const& col) : linked_column_view(nullptr, col) {}
+  linked_column_view(column_view const& col);
```

Comment:
Recommend to remove the empty lines in this class, such as the ones right below this.